src/server.js

```diff
@@ -30,15 +30,45 @@ class Server {
 
     async start () {
 
-        const listenPromise = once(this._server, 'listening');
+        //NOTE: getting settings
         const { token, owner, repo } = Config.globals;
+
+        //NOTE: getting data from github
         const apiGithubObj = new ApiGithub(token);
-        const listWFRuns = await apiGithubObj.getListRuns(owner, repo);
-        const runsKey = 'workflow_runs';
+        const listWFRuns   = await apiGithubObj.getListRuns(owner, repo);
+        const runsKey      = 'workflow_runs';
 
         debugApp(`Runs count: ${listWFRuns[runsKey] && listWFRuns[runsKey].length}`);
 
-        this._app.use(KoaStatic(path.resolve() + '/public'));
+        //NOTE: rendering output page
```

Comment:
It's good that you tried to provide some working example, but it's better to do not do this in your production code. Event if it is in a part of PR series that will be removed and refactored anyway. Just write a test that provides some dummy model (placeholder data) to your template, and assert the server's response.